assets/badger.txt

```diff
@@ -0,0 +1,26 @@
+                                   .oys:
+                                 .:dmmmmmmmddhyo+:.
+                              -oddy+:-.`` ``..:/ohmdy+-
+                            -hmmyosyyhhhhyyso+:.   -+ymd+`
+                       :ydmmmmmmmmmmmmmmmmmmmmmmds`  -hmmd.
+                      +mmmmmmmmmmmmmmmmmdhyso++/:.   smmmd.
+                      ymmmmmmmmmdyo/:.`           `:odmy/
+                      -dmmmmh+-`             `-/ohmds:`
+                        hms.          `-/oshdmmmmmms
+                       -mh        ./shmmmmmmmmmmmmmm+
+                       sm:     .ohmmmmmmmmmmmmmmmmmmm:
+                       hd`   :ymmmmmmmmmmmmmmmmmmmmmmm:
+                      `dh  .ymmmmmmmmmmmmmmmmmmmmmmmmmmo`
+                      `dh :mmmmmmmmmmmmmmmmmmmmmmmmmmmmmms:`
+                       hmsmmmmmmmmmmmmmmmmmmmmmmmmmmmmmhss+.
+                       ommmmmmmmmmmmmmmmmmmmmmmmmmmmmmd`
+                       .mmmmmmmmmmmmmmmmmmmmmmmmmmmmmm/
+                        ommmmmmmmmmmmmmmmmmmmmmmmmmmm/
+                        `ymmmmmmmmmmmmmmmmmmmmmmmmmy.
+                         `ymmmmmmmmmmmmmmmmmmmmmmmy
+                           +mmmmmmmmm/::mmmmmmmmms`
+                            .smmmmmmm- .mmmmmmmd:
+                              `/osyys. `oyyyso-
+
+                              Red Badger Website.
+                                   Honestly.
```

Comment:
I think we should keep this ASCII art, but change ASCII art on BB and React.London to something different.